phys2bids/physio_obj.py

```diff
@@ -325,6 +325,79 @@ def print_info(self, filename):
 
         LGR.info(info)
 
+    def check_trigger_amount_std(self, chtrig=1, num_timepoints_expected=0, tr=0):
+        from itertools import groupby
```

Comment:
I would move this import to the beginning of the file - itertools is a core module, so we know it will be installed, without adding specific dependencies, when phys2bids is installed!

Reply:
Sure, I was putting it there for the moment because I tought it wasn't being used by any other function